website/home/app/showcase/showcase-client.tsx

```diff
@@ -0,0 +1,389 @@
+"use client";
+
+import { motion } from "framer-motion";
+import Image from "next/image";
+import Link from "next/link";
+import AnimatedTitle from "@/components/ui/AnimatedTitle";
+import { ExternalLink } from "lucide-react";
+
+// Showcase data structure - easy to add new items
+interface ShowcaseItem {
+  id: string;
+  title: string;
+  description: string;
+  image?: string;
+  demo?: string; // Link to live demo
+  github?: string; // Link to GitHub repo
+  tags: string[];
+}
+
+const showcaseItems: ShowcaseItem[] = [
+  {
+    id: "chat-ux",
+    title: "Chat UX Template",
+    description:
+      "Next.js chat interface with real-time streaming responses using GenSX workflows and server-sent events.",
+    demo: "https://gensx-chat-ux.vercel.app",
+    github: "https://github.com/gensx-inc/gensx/tree/main/examples/chat-ux",
+    tags: ["Framework: Next.js", "Feature: Streaming", "Type: UI Template"],
+  },
+  {
+    id: "draft-pad",
+    title: "Draft Pad",
+    description:
+      "Document editor with AI-powered writing assistance, supporting real-time suggestions and markdown export.",
+    demo: "https://draft-pad.vercel.app",
+    github: "https://github.com/gensx-inc/gensx/tree/main/examples/draft-pad",
+    tags: [
+      "Framework: Next.js",
+      "Category: Writing Tool",
+      "Feature: Real-time",
+    ],
+  },
+  {
+    id: "deep-research",
+    title: "Deep Research Agent",
+    description:
+      "Multi-step research workflow that searches ArXiv papers, extracts key findings, and generates structured summaries.",
+    github:
+      "https://github.com/gensx-inc/gensx/tree/main/examples/deep-research",
+    tags: [
+      "Category: Research Agent",
+      "Integration: ArXiv API",
+      "Type: Workflow",
+    ],
+  },
+  {
+    id: "blog-writer",
+    title: "Blog Writer",
+    description:
+      "Workflow that generates blog posts by researching topics, creating outlines, and writing SEO-optimized content.",
+    github: "https://github.com/gensx-inc/gensx/tree/main/examples/blog-writer",
+    tags: [
+      "Category: Content Generation",
+      "Type: Workflow",
+      "Feature: Multi-step",
+    ],
+  },
+  {
+    id: "hacker-news-analyzer",
+    title: "Hacker News Analyzer",
+    description:
+      "Fetches and analyzes Hacker News posts and comments to identify trends and sentiment patterns.",
+    github:
+      "https://github.com/gensx-inc/gensx/tree/main/examples/hacker-news-analyzer",
+    tags: [
+      "Category: Analytics",
+      "Integration: HN API",
+      "Type: Data Processing",
+    ],
```

Comment:
I'm a bit underwhelmed by all the tags. I wonder if we should just remove them for now until they actually have utility (users can filter on them and we have discrete categories)

Reply:
Agreed. Removed for now but capability is still there when we decide to use it.